### DISCUSSION TITLE
Show your ForceFeedback Settings

### DISCUSSION THREAD

**Discussion by liszca** [gotzl/hid-fanatecff#52] [General]
Let me start!

My current Force Feedback settings on Fanatec CSL DD & Clubsport Pedals V3.

Note: Currently after turning on my wheel I have to cycle threw all modes and get back to the mode with the red light.

My Rating describes if I have the feeling Force Feedback is correct or at least very close to the Windows solution. I am **not** rating the games Force Feedback, there RRRE is unbeaten by a considerable margin.

Edit: ACC and RRRE need completely different settings for me, otherwise ACC feels wired.
As can be seen, I prefer Linear FF scaling.
# RRRE
Rating Force Feedback Quality: 5/5 (Very Sure you get what the Developers  wanted to deliver)
## Fanatec CSL DD Settings
* FF Str.  100
* FF Scale Linear
* Nat.Dpr. 40
* Nat.Fri. 60
* Nat.Inert. 40
* FF Interp. 12
* FE Intens. 100
* For. Eff.% 100
* Spr.Eff.% 100
* Dpr.Eff.% 100
* Brk.Ind. 85
* MPS Mo. Encoder
* Brk.For. 50

## Ingame
In general I followed the guide ingame under Force Feedback
for more immersion change was:
* Stationary Friction: 100%
* Bump Amplification: 33%

# ACC
Rating Force Feedback Quality: 5/5 (Very Sure you get what the Developers  aiming/able to deliver)

## Fanatec CSL DD Settings
* FF Str.  100
* FF Scale Linear
* Nat.Dpr. OFF
* Nat.Fri. OFF
* Nat.Inert. OFF
* FF Interp. OFF
* FE Intens. 100
* For. Eff.% 100
* Spr.Eff.% 100
* Dpr.Eff.% 100
* Brk.Ind. 85
* MPS Mo. Encoder
* Brk.For. 50

## Ingame
* Gain: 70% (lower this if using Peak scaling)
* Minimum Force: 0%
* Damper: 10%
* Dynamic Damping: 100%
* Road Effects: 0%
* Frequency: 400Hz

And it feels great! But comments are welcome.

Under Windows I used to use completely different settings like the ones in their Fanatec forum.

**Comment by ciscoyo** [comment]
i think i raced against u in rre.. your name looks kinda familiar.. ^^

> Under Windows I used to use completely different settings like the ones in their Fanatec forum.

man.. we r searching for guy like u the howl frikin time! finally a windows comparsion! we need to know if there r any differences (NOT in the ffb strengh) compare to this driver here! we r highly appreciate ur imput on this! ^^

#-- i will edit later for my settings

**Comment by liszca** [comment]
> man.. we r searching for guy like u the howl frikin time! finally a windows comparsion! we need to know if there r any differences (NOT in the ffb strengh) compare to this driver here! we r highly appreciate ur imput on this! ^^

In comparison to Windows there is a slight difference, but I don't have a Windows installation any more. 

Some bugs I have adjusting settings at the wheel while in Game. 
ACC: It can happen that the driver looks to the right --> fix hit "4" or "6" on the numpad, so not really an issue.

RRRE: doing the same adjustment can be very upsetting, it can be a simple Breakbalance change or the pit menu gets opened and something gets changed and the engine can be turned off.

How can I calibrate my Breakpadels, right now I don't need too, but how are you doing it?

**Reply by SynthetikzZ** [reply]
Currently I'm not at home, tomorrow I will do a complete USB dump from the fanatec driver ( includes calibrating wheel center/ pedals etc) and to dump friction and inertia effects for gotzl.

**Comment by liszca** [comment]
> Some bugs I have adjusting settings at the wheel while in Game. ACC: It can happen that the driver looks to the right --> fix hit "4" or "6" on the numpad, so not really an issue.
> 
> RRRE: doing the same adjustment can be very upsetting, it can be a simple Breakbalance change or the pit menu gets opened and something gets changed and the engine can be turned off.

Looks like those problems are gone with the current version.

**Reply by ciscoyo** [reply]
no sure what u mean with that, but ye, there **WAS** a bug in rre.
when i go into the wheel ffb menu and changing things, all buttons were automaticly pressed/triggered and holded.
i just need to press all button again, than it stops.
but with new update, this behavior is gone ^^

> How can I calibrate my Breakpadels, right now I don't need too, but how are you doing it?

again, im not sure what u mean tbh.. i just press the breakpeddal and its automaticly calibrated(ingame option menu top screen u can see the amount of input with break-,gas-,clutchpedal and wheel rotation). same with the break keybind ^^
